jira.go

```diff
@@ -74,6 +77,9 @@ func openJiraTicket(endpointAPI string, orgID string, token string, jiraProjectI
 	jiraTicket.Fields.Projects.ID = jiraProjectID
 	jiraTicket.Fields.IssueTypes.Name = jiraTicketType
 	jiraTicket.Fields.Assignees.ID = assigneeID
+
+	emptyByteVar := make([]byte, 128)
```

Comment:
What is this var for?

Reply:
Testing if the the reply is empty and return an error.